discovery-provider/nginx_conf/nginx.conf

```diff
@@ -84,9 +101,14 @@ http {
         location ~* .*/search/.* {
             access_by_lua_block {
                 local main = require "main"
+                ngx.shared.request_count:incr("request-count", 1, 0)
                 return main.limit_to_rps()
             }
 
+            log_by_lua_block {
+                ngx.shared.request_count:incr("request-count", -1, 0)
```

Comment:
perhaps we can name the increment/decrement `mark_request_processing` and `mark_request_processed` since it's reused and that's what it appears to be doing?